File: sdk/r/R/duoc_pham_and_y_te.R
#' Dược phẩm & Y tế
#' @param http HttpClient object
#' @return Resource object
#' @export
DuocPhamAndYTeResource <- function(http) {
  env <- new.env(parent = emptyenv())
  env$http <- http
  class(env) <- "DuocPhamAndYTeResource"
  env
}

#' GET /api/health/drug/prices - Tra các lượt kê khai giá bán buôn thuốc dự kiế...
#' @param self Resource object
#' @param ... Additional parameters
#' @return API response
healthDrugPrices <- function(self, ...) {
  self$http$get("/api/health/drug/prices", list(...))
}

#' GET /api/health/drug/tenders - Tra kết quả trúng thầu thuốc tại các cơ sở y t...
#' @param self Resource object
#' @param ... Additional parameters
#' @return API response
healthDrugTenders <- function(self, ...) {
  self$http$get("/api/health/drug/tenders", list(...))
}

#' GET /api/health/drugs - Tìm trong sổ đăng ký thuốc được phép lưu hành...
#' @param self Resource object
#' @param ... Additional parameters
#' @return API response
healthDrugs <- function(self, ...) {
  self$http$get("/api/health/drugs", list(...))
}

#' GET /api/health/drugs/2 - Hồ sơ đầy đủ của MỘT SỐ ĐĂNG KÝ
#' @param self Resource object
#' @param ... Additional parameters
#' @return API response
healthDrugs2 <- function(self, ...) {
  self$http$get("/api/health/drugs/2", list(...))
}

#' GET /api/health/status - Độ tươi của ba bản chụp trên máy chủ này: số b...
#' @param self Resource object
#' @param ... Additional parameters
#' @return API response
healthStatus <- function(self, ...) {
  self$http$get("/api/health/status", list(...))
}

File: sdk/c/src/duoc_pham_and_y_te.c
#include "hitechcloud/duoc_pham_and_y_te.h"
#include <stdlib.h>


/* GET /api/health/drug/prices - Tra các lượt kê khai giá bán buôn thuốc dự kiế... */
char *hc_duoc_pham_and_y_te_health_drug_prices(hc_http_client_t *client) {
    return hc_get(client, "/api/health/drug/prices", NULL);
}

/* GET /api/health/drug/tenders - Tra kết quả trúng thầu thuốc tại các cơ sở y t... */
char *hc_duoc_pham_and_y_te_health_drug_tenders(hc_http_client_t *client) {
    return hc_get(client, "/api/health/drug/tenders", NULL);
}

/* GET /api/health/drugs - Tìm trong sổ đăng ký thuốc được phép lưu hành... */
char *hc_duoc_pham_and_y_te_health_drugs(hc_http_client_t *client) {
    return hc_get(client, "/api/health/drugs", NULL);
}

/* GET /api/health/drugs/2 - Hồ sơ đầy đủ của MỘT SỐ ĐĂNG KÝ */
char *hc_duoc_pham_and_y_te_health_drugs_2(hc_http_client_t *client) {
    return hc_get(client, "/api/health/drugs/2", NULL);
}

/* GET /api/health/status - Độ tươi của ba bản chụp trên máy chủ này: số b... */
char *hc_duoc_pham_and_y_te_health_status(hc_http_client_t *client) {
    return hc_get(client, "/api/health/status", NULL);
}

